src/test/java/petStoreTests/user/AddUserTest.java

```diff
@@ -0,0 +1,35 @@
+package petStoreTests.user;
+
+import client.UserClient;
+import data.DataSet;
+import dto.requests.ResponseInfo;
+import dto.requests.user.User;
+import io.qameta.allure.Description;
+import org.apache.http.HttpStatus;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import petStoreTests.AbstractTest;
+
+import java.io.IOException;
+
+public class AddUserTest extends AbstractTest {
+    @Description("Verify that we are adding user to the store data base")
+    @Test
+    public void addUserTest() throws IOException {
+        LOGGER.info("START TEST add user to the store data base");
+        User user = DataSet.addingUser();
+        ResponseInfo response = UserClient.postUser(user);
+
+        Assert.assertEquals(response.getCode(), HttpStatus.SC_OK);
+        Assert.assertEquals(response.getType(), DataSet.messageUnknownResponse().getType());
+        Assert.assertEquals(response.getMessage(), String.valueOf(user.getId()));
+
+        User newUser = UserClient.getUserByUsername(user.getUsername());
+        Assert.assertEquals(newUser.getId(), user.getId());
```

Comment:
Why not assert that both `user` and `newUser` objects are equal, and not only their usernames?